.github/workflows/master.yml

```diff
@@ -32,6 +32,7 @@ env:
   PIP_CONSTRAINT: ${{ github.workspace }}/requirements/constraints.txt
   PYTHONUTF8: "1"
   _MLFLOW_TESTING_TELEMETRY: "true"
+  UV_PRERELEASE: "allow"
```

Comment:
Do we need this during test?

Reply:
`uv run --no-project --with mlflow-skinny==3.2.0rc0 mlflow --version` actually worked fine, which implies direct dependencies (`mlflow-skinny==3.2.0rc0`) can be pre-released versions.

Reply:
Ah ok, then I can remove this :)